### DISCUSSION TITLE
the use of simAddVehicle()

### DISCUSSION THREAD

**Discussion by chenzhoule-6** [microsoft/AirSim#4490] [Support Q&A]
How to generate your own model through the API simaddvehicle(),for example, people or animals, rather than the default cars and drones?

**Comment by zimmy87** [comment]
First, follow this tutorial for defining a custom vehicle blueprint: https://youtu.be/Bp86WiLUC80 (this one is for a custom multirotor, but you should be able to use the same technique on `Blueprint'/AirSim/VehicleAdv/SUV/SuvCarPawn.SuvCarPawn'`). Once you have your custom vehicle blueprint that references your own model, you can define a [PawnPath](https://microsoft.github.io/AirSim/settings/#pawnpaths) in your settings, then pass the name of that PawnPath as the last argument to your call to the simAddVehicle() API.

**Reply by chenzhoule-6** [reply]
I have made many attempts, but I can't achieve the desired effect, I can't replace it with the character model I need, and control it through the API, I don't know whether there is a more detailed process.

**Reply by Yasaman-Haghighi** [reply]
Did you find any solution for this?
